caracal/sample_configurations/carateConfig.yml

```diff
@@ -111,24 +111,36 @@ inspect__2:
     enable: true
     default_column: CORRECTED_DATA
     plots_by_field:
-      - "-x real  -y imag   -c CORR"
-      - "-x real  -y imag   -c SCAN_NUMBER"
-      - "-x UV    -y amp    -c SCAN_NUMBER"
+      - "-x BASELINE_M -y FREQ -c amp"
+      - "-x real  -y imag   -c SCAN_NUMBER --cnum 100"
+      - "-x real  -y imag   -c ANTENNA1 --cnum 100"
+      - "-x ANTENNA1 -y ANTENNA2 -c SCAN_NUMBER --cnum 100 --aaxis phase --ared std"
+      - "-x ANTENNA1 -y ANTENNA2 -c SCAN_NUMBER --cnum 100 --aaxis amp --ared mean"
+      - "-x UV    -y amp    -c SCAN_NUMBER --cnum 100"
       - "-x UV    -y amp    -c ANTENNA1 --cnum 64"
-      - "-x UV    -y phase  -c SCAN_NUMBER"
       - "-x UV    -y phase  -c ANTENNA1 --cnum 64"
-      - "-x FREQ  -y amp    -c SCAN_NUMBER"
+      - "-x FREQ  -y amp    -c SCAN_NUMBER --cnum 100"
       - "-x FREQ  -y amp    -c ANTENNA1 --cnum 64"
-      - "-x FREQ  -y phase  -c SCAN_NUMBER"
-      - "-x FREQ  -y phase  -c ANTENNA1 --cnum 64"
       - "-x U     -y V      -c amp"
-      - "-x U     -y V      -c phase --cnum 64 --cmin -5 --cmax 5"
-      - "-x BASELINE -y FREQ -c amp"
-      - "-x BASELINE -y FREQ -c CORRECTED_DATA-MODEL_DATA:amp"
-    plots_by_corr:
-      - "-x real  -y imag   -c FIELD_ID"
+      - "-x U     -y V      -c phase --cmin -5 --cmax 5"
+    plots:
+      - "-x real  -y imag   -c SCAN_NUMBER --cnum 100 --col CORRECTED_DATA/MODEL_DATA --iter-field"
+      - "-x real  -y imag   -c ANTENNA1 --cnum 100 --col CORRECTED_DATA/MODEL_DATA --iter-field"
+#      - '-x FREQ -y phase --ymin -40 --ymax 40 -c SCAN_NUMBER --cnum 100 --field {bpcal} --dir {msbase}-bpcal-byant --iter-ant'
```

Comment:
Might I suggest making a short article on Wiki instead of having all the commented stuff in the config files? Seems better aesthetically. All these commands for shadems are very handy, but maybe we could have a quick explanation of what they mean for some of the more unusual ones (comes in handy for super-lazy users like me).